src/components/Row/Row.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person
+  onSelectPerson: (slug: string) => void
+  isSelected: boolean
+  motherSlug?: string
+  fatherSlug?: string
+};
+
+export const Row: React.FC<Props> = ({
+  person,
+  onSelectPerson,
+  isSelected,
+  motherSlug,
+  fatherSlug,
+}) => {
+  const {
+    slug, motherName, sex, born, died, name, fatherName,
+  } = person;
+
+  return (
+    <tr
+      data-cy="person"
+      className={
+        isSelected ? 'has-background-warning' : undefined
```

Comment:
better to use className library to make a dynamic className 
please check other places